### ISSUE TITLE
Quoted commands fail

### ISSUE THREAD

**Issue by brandondrew** [mklement0/ttab#7]
Certain commands are not (AFAIK) possible with ttab, because of precedence rules.
~~~
ttab -t 'code & git' (git branch; git status)
ttab -t 'code & git' (git branch && git status)
~~~

It might seem that putting quotes around the command would fix this, passing it all to ttab as a single argument that could be unpacked before being issued to the shell, but it does not work.

Allowing the command and its arguments to be either single or double quoted (but not requiring it!) would be very helpful, as it would allow commands like those given above to succeed.

**Comment by mklement0** [comment]
The `man` page states (emphasis added):

--

To specify _multiple_ commands, use `eval` _followed by a single, quoted
       string_ containing the entire shell command line to execute; in the simplest
       case, enclose the string in single-quotes and use `;` to separate  com-
       mands.

--

Thus, you'd have to specify

     ttab -t 'code & git' eval 'git branch && git status'

**Comment by mklement0** [comment]
I realized that I could make the `eval` implicit, so if you install the just-released [v0.5.0](https://github.com/mklement0/ttab/releases/tag/v0.5.0)), you can omit it:

     ttab -t 'code & git' 'git branch && git status'

In other words: you can now pass any shell command line as a _single, quoted string_ as the one and only operand (non-option)  to `ttab` to have it executed in the new tab.

Note that quoting the command line is _invariably_ a requirement, as the _current_ shell would otherwise interpret metacharacters such as `;`, `(`, `)` and `&`, before `ttab` ever gets to see the command.
